src/components/TransactionImport/Result.tsx

```diff
@@ -164,6 +165,22 @@ const Result = (props: Props) => {
     setTransactions(tmpTransactions)
   }
 
+  const updateValues = (
+    values: { key: keyof UnpersistedTransaction; value: any }[]
+  ) => {
+    const tmpTransactions = [...transactions]
+
+    const newTransaction = { ...tmpTransactions[currentIndex].transaction }
+    values.forEach(
+      (value: { key: keyof UnpersistedTransaction; value: any }) => {
```

Comment:
The type definition is unnecessary here, since typescript already knows the type of `values`.